### PR TITLE
Set params when loading nvimcom

### LINKED ISSUE #294: R terminal crash on qmd  open or  leader RF
Hi 

In the last couple of days I have been having difficulties working with quarto files on windows.

When I select a QMD file and then initiate an R terminal the following error is returned:

Warning: stack imbalance in '::', 71 then 69
Warning: stack imbalance in '$', 68 then 66 
[Process exited -1073741784]

If I first open an R file I am able to initialise the R terminal.

But when I then open a Quarto file I get the following error

> Error: '\R' is an unrecognized escape in character string (<input>:1:29)

[Process exited -1073741784]

I am happy to provide additional information if you can let me know what would be useful.

Many thanks.

### PR DESCRIPTION
**Pull request by jalvesaq**
Close #294

### CODE REVIEW

**Comment by jalvesaq** [comment]
This pull request sets the value of the params object while loading nvimcom to avoid a crash on Windows.

**Comment by jalvesaq** [comment]
@jasonpott, the R code to create the params object now is called only when there is any change in the params field which is safer than running it whenever the document is saved.

@idavydov may also want to review this pull request...

File: lua/r/rmd.lua [hunk 1]
```diff
@@ -147,47 +147,64 @@ M.next_chunk = function()
...
         if lines[i] == "params:" then
-            p = true
-            break
+            local cp = ""
+            i = i + 1
+            while i < #lines and lines[i]:sub(1, 1) == " " do
+                cp = cp .. lines[i]
+                i = i + 1
+            end
+            if last_params == cp then return "old" end
```
Comment (PMassicotte) [inline lua/r/rmd.lua:171]: Is `old` ever used elsewhere?

**Comment by jalvesaq** [comment]
> Is old ever used elsewhere?

No. It has the same practical effect as "none": _Don't run any R code_. However, "none" wouldn't correspond to the reality when there is already an old params field. Perhaps we could return "unchanged" for both cases.

**Comment by PMassicotte** [comment]
Other than that, I tried on my Linux machine and it works.

**Comment by PMassicotte** [comment]
Thanks for this @jalvesaq

**Comment by jalvesaq** [comment]
> If it would be helpful I'm happy to test future versions on windows if that would help the project?

Yes, thank you!

**Pull request merged**